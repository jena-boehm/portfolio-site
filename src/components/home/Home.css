.homeContainer {
    display: flex;
    justify-content: center;
    align-items: center;
    height: 90vh;
    width: 100%;
    background-color: #000000;
    background-image: url('../../../public/images/home-graphic.png');
    background-size: 100%;
    background-repeat: no-repeat;
    background-position: center;
    margin: 2.6em 0 2em 0;
    box-shadow: inset 0 0 50px 40px rgb(0, 0, 0);

    /* .homeGraphic {
        margin: 1em;
        box-shadow: 0 0px 8px 0px rgba(255,255,255, 0.2);
        background-blend-mode: multiply;
    } */
}