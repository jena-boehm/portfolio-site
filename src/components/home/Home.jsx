import React from 'react';
import styles from './Home.css';
// import homeGraphic from '../../../public/images/home-graphic.png';

export const Home = () => {
  return (
    <div className={styles.homeContainer}>
      {/* <img 
        className={styles.homeGraphic} 
        src={homeGraphic} 
        alt="Jena Lux Boehm - Software Engineer"
        height="90%" 
        width="90%"
      /> */}
    </div>
  );
};
