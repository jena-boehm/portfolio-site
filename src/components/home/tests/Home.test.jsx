import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { Home } from '../Home';

describe('Home component', () => {
  afterEach(() => cleanup());
  test.skip('renders Home', () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });
});
