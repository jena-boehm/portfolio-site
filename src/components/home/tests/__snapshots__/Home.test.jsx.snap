// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders Home 1`] = `
<DocumentFragment>
  <div
    class="homeContainer"
  >
    <img
      alt="Jena Lux Boehm - Software Engineer"
      class="homeGraphic"
      height="90%"
      src="/path/image"
      width="90%"
    />
  </div>
</DocumentFragment>
`;
